pkg/autoscaler/statserver/server_test.go

```diff
@@ -157,8 +156,12 @@ func assertReceivedOk(sm *autoscaler.StatMessage, statSink *websocket.Conn, stat
 	if !ok {
 		t.Fatalf("statistic not received")
 	}
-	if !cmp.Equal(sm, recv) {
-		t.Fatalf("Expected and actual stats messages are not equal: %s", cmp.Diff(sm, recv))
+	if recv.Stat.Time == nil {
+		t.Fatalf("Stat time is nil")
+	}
+	ignoreTimeField := cmpopts.IgnoreFields(autoscaler.StatMessage{}, "Stat.Time")
+	if !cmp.Equal(sm, recv, ignoreTimeField) {
+		t.Fatalf("Expected and actual stats messages are not equal: %s", cmp.Diff(sm, recv, ignoreTimeField))
```

Comment:
Can you add a notice on how to read the diff here please? Like in

```golang
t.Logf("For service %s traffic shape mismatch: (-got, +want) %s", sName, cmp.Diff(got, want))
```

Reply:
change to 
```
t.Fatalf("StatMessage mismatch: diff (-got, +want) %s", cmp.Diff(recv, sm, ignoreTimeField))
```
Is that ok?